presto-main/etc/log.properties

```diff
@@ -9,3 +9,4 @@ com.facebook.presto=INFO
 com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory=WARN
 com.ning.http.client=WARN
 com.facebook.presto.server.PluginManager=DEBUG
+com.facebook.presto.server.security.LdapAuthenticator=DEBUG
```

Comment:
Let's remove this, as it's not broadly useful for development. We expect people to add things as needed while developing, without having everything enabled by default.